tests/db_tests/conftests.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 QHAna plugin runner contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the db module of the plugin_utils."""
+
+from os import environ
+import pytest
+from dotenv import load_dotenv
+
+load_dotenv(".flaskenv")
+load_dotenv(".env")
+
+MODULE_NAME = "qhana_plugin_runner"
+
+from qhana_plugin_runner import create_app
+from qhana_plugin_runner.db.cli import create_db_function, drop_db, create_db
+
+from qhana_plugin_runner.db.models.tasks import ProcessingTask
+from qhana_plugin_runner.db.models.mutable_json import (
+    NestedMutableDict,
+    NestedMutableList,
+)
+
+
+@pytest.fixture(scope="function")
+def task_data():
+
+    environ["SQLALCHEMY_DATABASE_URI"] = "sqlite:///../instance/test.db"
```

Comment:
There is also the option for an in memory sqite DB. Because a test DB should always be in a defined state at test start either create the DB in a temporary folder (use https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory as a context manager for automatic cleanup) or make sure to clean up the test database completely (e.g. delete the sqlite file)